<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QWebEngineView" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>0</y>
          <width>881</width>
          <height>471</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>691</width>
          <height>361</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Threads">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Threads</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_CPU">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Memory">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Memory</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Media">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>170</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Media</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>What kind of metric do you pefer ?</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>pid of process to kill ?</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_kill">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>kill</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>113</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QCustomPlot" name="chartOfCPU" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1131</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="chartOfMem" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>1131</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>memory usage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cpu usage</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QCustomPlot" name="chartOfEnergyPercentage" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>551</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="chartOfEnergyTime" native="true">
        <property name="geometry">
         <rect>
          <x>579</x>
          <y>20</y>
          <width>561</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
